README.md

```diff
@@ -54,11 +64,13 @@ The `connect` event will fire only once when the initial ssh connection is made.
 - `kill` - a method to kill autossh
 - `pid` - the autossh process id
 - `host`
-- `localHost` - The host, to which the tunnel applies.
+- `localHost` - the host, to which the tunnel applies
 - `username`
 - `remotePort`
 - `localPort`
-- `execString` - the autossh command string
+- `useAutossh` - use autossh (see https://linux.die.net/man/1/autossh)
+- `executable` - the ssh connabd to use (it is passed to `/usr/bin/env`, **Higly Optional!**)
```

Comment:
what is `connabd `?

Reply:
Looking at my keyboard, I'm assuming it was a typo for command

Reply:
yes